command/job_status_test.go

```diff
@@ -26,6 +26,21 @@ func TestJobStatusCommand_Run(t *testing.T) {
 	t.Parallel()
 	srv, client, url := testServer(t, true, nil)
 	defer srv.Shutdown()
+	testutil.WaitForResult(func() (bool, error) {
+		nodes, _, err := client.Nodes().List(nil)
+		if err != nil {
+			return false, err
+		}
+		if len(nodes) == 0 {
+			return false, fmt.Errorf("missing node")
+		}
+		if _, ok := nodes[0].Drivers["mock_driver"]; !ok {
+			return false, fmt.Errorf("mock_driver not ready")
+		}
+		return true, nil
+	}, func(err error) {
+		t.Fatalf("err: %s", err)
+	})
```

Comment:
This snippet seems duplicated few times (with slight modifications `fs_test.go` -- should we extract a `waitUntilServerIsReady(server)` function?  Would be handy for the e2e tests as well.  Alternatively, `testServer` (and e2e equivalent helper function) can block until client is ready to avoid waiting boilerplate.